diff --git a/jsx-namespace.d.ts b/jsx-namespace.d.ts
index b9dd5a457f94e75a70321be3a0231f30a2aead82..19253f5b1f48b200cf4af6d4f1bbd87a2dc3b308 100644
--- a/jsx-namespace.d.ts
+++ b/jsx-namespace.d.ts
@@ -36,7 +36,7 @@ export namespace JSX {
 
   interface IntrinsicAttributes extends React.Attributes {}
 
-  interface IntrinsicElements extends React.JSX.IntrinsicElements, ExtendedIntrinsicElements<OpenTUIComponents> {
+  interface IntrinsicElements extends ExtendedIntrinsicElements<OpenTUIComponents> {
     box: BoxProps
     text: TextProps
     span: SpanProps
