diff --git a/jsx-namespace.d.ts b/jsx-namespace.d.ts
index c9148fdcbc345b2511dd8f68cd1a0003e6d33f7a..e21d11749c87afe38af1a0a18b854ce3e9bfaeb2 100644
--- a/jsx-namespace.d.ts
+++ b/jsx-namespace.d.ts
@@ -37,7 +37,7 @@ export namespace JSX {
 
   interface IntrinsicAttributes extends React.Attributes {}
 
-  interface IntrinsicElements extends React.JSX.IntrinsicElements, ExtendedIntrinsicElements<OpenTUIComponents> {
+  interface IntrinsicElements extends ExtendedIntrinsicElements<OpenTUIComponents> {
     box: BoxProps
     text: TextProps
     span: SpanProps
